content/en/docs/concepts/overview/working-with-objects/labels.md

```diff
@@ -61,12 +64,12 @@ Via a _label selector_, the client/user can identify a set of objects. The label
 The API currently supports two types of selectors: _equality-based_ and _set-based_.
 A label selector can be made of multiple _requirements_ which are comma-separated. In the case of multiple requirements, all must be satisfied so the comma separator acts as a logical _AND_ (`&&`) operator.
 
-An empty label selector (that is, one with zero requirements) selects every object in the collection.
-
-A null label selector (which is only possible for optional selector fields) selects no objects.
+The semantics of empty or non-specified selectors are dependent on the context,
+and API types that use selectors should document the validity and meaning of
+them.
 
 {{< note >}}
-The label selectors of two controllers must not overlap within a namespace, otherwise they will fight with each other.
+For some API types, such as ReplicaSets, the label selectors of two instances must not overlap within a namespace, or the controller will fight with itself.
```

Comment:
nit: Suggestion, either explain `or the controller will fight with itself` or drop it altogether.

Reply:
Fixed in repush